LICENSE

```diff
@@ -1,22 +1,22 @@
 BSD 3-Clause License
 
-Copyright (c) 2021, 2022 Chudi Zhong, Hayden McTavish, Jimmy Lin, Margo Seltzer,
-Cynthia Rudin, The University of British Columbia, Duke University
+Copyright (c) 2021, 2022, 2024, Chudi Zhong, Hayden McTavish, Jimmy Lin, Ilias Karimalis,
+Margo Seltzer, Cynthia Rudin, The University of British Columbia, Duke University
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are met:
 
-1. Redistributions of source code must retain the above copyright notice, this
-   list of conditions and the following disclaimer.
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
 
-2. Redistributions in binary form must reproduce the above copyright notice,
-   this list of conditions and the following disclaimer in the documentation
-   and/or other materials provided with the distribution.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
 
-3. Neither the name of the copyright holder nor the names of its
-   contributors may be used to endorse or promote products derived from
-   this software without specific prior written permission.
+* Neither the name of the vector package developers nor the names of its
```

Comment:
This line we might want to leave as copyright holders/developers, right?

Reply:
This seems to be a result of the Python project template this was built on top of. Reversing the change now!